### DISCUSSION TITLE
How to use flake8 for python diagnostics?

### DISCUSSION THREAD

**Discussion by parthokr** [nvimtools/none-ls.nvim#104] [Q&A] [answered]
I don't see any builtins for flake8.
Is there any workaround?

**Comment by mochaaP** [comment, marked as answer]
It's in none-ls-extras. :)

https://github.com/nvimtools/none-ls-extras.nvim/blob/main/lua/none-ls/diagnostics/flake8.lua

**Reply by parthokr** [reply]
thanks @mochaaP
